src/pypi_attestations/_impl.py

```diff
@@ -331,3 +331,85 @@ def _ultranormalize_dist_filename(dist: str) -> str:
         return f"{name}-{ver}.tar.gz"
     else:
         raise ValueError(f"unknown distribution format: {dist}")
+
+
+class Publisher(BaseModel):
+    """Publisher as defined in PEP 740."""
+
+    kind: str
+    """
+    The kind of Trusted Publisher.
+    """
+
+    claims: Union[object, None] = Field(...)  # noqa: UP007
+    """
+    Claims specified by the publisher.
+    """
+
+    @classmethod
+    def from_kind(cls, kind: str) -> Publisher:
+        """Construct a Publisher from a kind."""
+        return Publisher(kind=kind, claims=None)
```

Comment:
Is there a reason why we need this constructor? I think we can just write `Publisher(kind=var, claims=None)`

Reply:
I've removed it